src/rollback_to_stable/rts_btree_walk.c

```diff
@@ -104,13 +104,14 @@ __rts_btree_walk_page_skip(
         __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
           WT_RTS_VERB_TAG_STABLE_PG_WALK_SKIP "ref=%p: stable page walk skipped", (void *)ref);
         WT_STAT_CONN_INCR(session, txn_rts_tree_walk_skip_pages);
-    }
+    } else {
+        reconciled = ref->page && ref->page->modify ? true : false;
```

Comment:
Is there any reason why we want to print the `reconciled` property only in this code path?

Reply:
in this flow, the page is on disk, we shouldn't have a `ref->page` itself.

Reply:
Removed in aeac5f015904e4bdd5ed7719deec3d004fc2c37e